catalog/core/catalog-core-api/src/main/java/ddf/catalog/CatalogFramework.java

```diff
@@ -380,6 +380,12 @@ SourceInfoResponse getSourceInfo(SourceInfoRequest sourceInfoRequest)
      * <li/>Return the last {@link QueryResponse} to the caller.
      * </ol>
      * </p>
+     * <p>
+     * <b>Important:</b> Implementations are free to limit the number of results returned
```

Comment:
@ricklarsen What would be your take on how to approach this in the documentation?